config/gitter-config-sample.yaml

```diff
@@ -0,0 +1,2 @@
+matrix_homeserver: "http://example.com:8008"
+matrix_user_domain: "example.com"
```

Comment:
The config schema YAML has got a bunch more than just this. It would be nice to put those other fields in the sample, even if they are commented out, so you can explain what they do (I for one have no idea what a `name_mangling` array will do!).